hibike/devices/BatteryBuzzer/disp_8.cpp

```diff
@@ -0,0 +1,201 @@
+//code that deals with displaying the data to the outside world.
+
+#include <SevenSeg.h>
+#include "disp_8.h"
+
+//Segment Pins, A toG then decimal pt.
+SevenSeg disp(8,9,7,2,0,15,14);
```

Comment:
Let's make a "Pindefs.h", and keep all pin-related information (Such as the pins used for the display, etc) there.  That'll make the code easier to read, and easier to change in the future.